### PR TITLE
Add hit test article.

### LINKED ISSUE #2090: content: WebXR Hit-Test
**Author**: @jpmedley 
**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]

**Target publish date:** 2020-02-13
- [ ] Check this box if this is a hard deadline.

- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

Check out the [web.dev writing process](https://web.dev/handbook/quick-start/).

When you're ready, make a copy of [this template](https://docs.google.com/document/d/1lgaNIEnXZf-RB8_p9RK22QEgpXJqnu77pLWVWVy4nuw/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [x] 1. I've done my [proposal](https://docs.google.com/document/d/1WkvSgSBCtG24BZsdBc5lFb0beuW8raWcxJYyQ3rJglA/edit)
- [ ] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [ ] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request #<add PR number>
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #2090

Changes proposed in this pull request:

- Adds an article on the WebXR Hit Test

### CODE REVIEW

**Comment by jpmedley** [comment]
@kaycebasques @mfriesenhahn Can one of you please review thias?

File: src/site/content/en/blog/ar-hit-test/index.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
+---
+title: Positioning virtual objects in real-world views
+subhead: The Hit Test API lets you position virtual items in a real-world view.
+authors:
+  - joemedley
+date: 2020-02-12
```
Comment (jpmedley) [inline src/site/content/en/blog/ar-hit-test/index.md]:
```suggestion
date: 2020-02-13
```

**Comment by jpmedley** [comment]
FYI, I've set draft to true. When this is merged, I'll open a separate PR to remove the draft tag from both this and the AR article, then they'll go together.

**Comment by jpmedley** [comment]
What needs to be done with the image? The handbook isn't clear.

**Comment by kaycebasques** [comment]
* The filename was `reticales.png` and it appears that you meant it to be `reticles.png` and therefore the image in the post was broken
* The image file was 1.6MB, running it through tinypng.com reduced it to 600KB

Just pushed a commit to fix, waiting on CI to pass

**Pull request merged**